uqcsbot/poll.py

```diff
@@ -0,0 +1,82 @@
+import discord
+from discord import app_commands
+from discord.ext import commands
+
+from typing import Any
+
+# The unicode points for each number emoji starting from 0 to 10
+EMOJI_NUMBERS = [
+    "\u0030\u20E3",
+    "\u0031\u20E3",
+    "\u0032\u20E3",
+    "\u0033\u20E3",
+    "\u0034\u20E3",
+    "\u0035\u20E3",
+    "\u0036\u20E3",
+    "\u0037\u20E3",
+    "\u0038\u20E3",
+    "\u0039\u20E3",
+    "\U0001F51F",
+]
+
+
+class PollMenu(discord.ui.Modal, title="New Poll"):
+    """The menu to create a new poll"""
+
+    poll_question: discord.ui.TextInput[Any] = discord.ui.TextInput(
+        label="A question of the poll",
+        placeholder="Your question goes here (e.g. What does UQCS stand for?)",
+        max_length=100,
+    )
+    poll_options: discord.ui.TextInput[Any] = discord.ui.TextInput(
+        label="The options for the poll (one per line)",
```

Comment:
Might be good to include the max 10 in the dialog, as you don't find out til after you put more than 10